src/models/AssetPrice.ts

```diff
@@ -67,35 +85,35 @@ export default class AssetRewards {
     /**
      * It adds network fees on top of the already configured asset rewards
      * @param feeReceiver - the address receiving the fees
-     * @param networkFeePercent - the percent of fees to receive, it uses the contract denomitor @see AssetRewards.NETWORK_FEE_DENOMINATOR
+     * @param networkFeePercent - the percent of fees to receive, it uses the contract denomitor @see AssetPrice.NETWORK_FEE_DENOMINATOR
      * @returns the asset rewards object
      */
     public addNetworkFees(
         feeReceiver: string,
         networkFeePercent: BigNumber
-    ): AssetRewards {
+    ): AssetPrice {
```

Comment:
I don't think we need to return anything

Reply:
method chaining